### PR TITLE
Update 2017

### PR DESCRIPTION
**Pull request by kdinev**

### CODE REVIEW

File: src/config/combo.json [hunk 1]
```diff
@@ -588,85 +516,6 @@
...
-    "requestType": {
-      "type": "string",
-      "defaultValue": "GET",
-      "description": {
-	  "en": "specifies the HTTP verb to be used to issue the request",
-	  "jp": "要求を送信する HTTP 動詞を指定します。"},
-      "valueOptions": null
-    },
-    "responseContentType": {
-      "type": "string",
```
Comment (bazal4o) [inline src/config/combo.json]: If the user setup uses remote datasource with remote url, is this property required?

File: src/config/combo.json [hunk 2]
```diff
@@ -588,85 +516,6 @@
...
-      "defaultValue": null,
-      "description": {
-	  "en": "content type of the response. See http://api.jquery.com/jQuery.ajax/ => contentType",
-	  "jp": "応答のコンテンツ タイプ。http://api.jquery.com/jQuery.ajax/ の contentType を参照してください。"},
-      "valueOptions": null
-    },
-    "responseDataKey": {
-      "type": "string",
```
Comment (bazal4o) [inline src/config/combo.json]: If the user setup uses remote datasource with remote url, is this property required?

File: src/config/combo.json [hunk 3]
```diff
@@ -588,85 +516,6 @@
...
-      "defaultValue": null,
-      "description": {
-	  "en": "see $.ig.DataSource. This is basically the property in the responses where data records are held, if the response is wrapped.",
-	  "jp": "$.ig.DataSource を参照してください。応答がラップされる場合、これは基本的にデータ レコードが保持される応答内のプロパティです。"},
-      "valueOptions": null
-    },
-    "responseDataType": {
```
Comment (bazal4o) [inline src/config/combo.json:623]: If the user setup uses remote datasource with remote url, is this property required?

File: src/config/combo.json [hunk 4]
```diff
@@ -588,85 +516,6 @@
...
-    },
-    "responseContentType": {
-      "type": "string",
-      "defaultValue": null,
-      "description": {
-	  "en": "content type of the response. See http://api.jquery.com/jQuery.ajax/ => contentType",
-	  "jp": "応答のコンテンツ タイプ。http://api.jquery.com/jQuery.ajax/ の contentType を参照してください。"},
-      "valueOptions": null
-    },
-    "responseDataKey": {
```
Comment (bazal4o) [inline src/config/combo.json:615]: If the user setup uses remote datasource with remote url, is this property required?

File: src/config/combo.json [hunk 5]
```diff
@@ -588,85 +516,6 @@
...
-    },
-    "requestType": {
-      "type": "string",
-      "defaultValue": "GET",
-      "description": {
-	  "en": "specifies the HTTP verb to be used to issue the request",
-	  "jp": "要求を送信する HTTP 動詞を指定します。"},
-      "valueOptions": null
-    },
-    "responseContentType": {
```
Comment (bazal4o) [inline src/config/combo.json:607]: If the user setup uses remote datasource with remote url, is this property required?

File: src/config/datachart.json [hunk 6]
```diff
@@ -498,22 +498,6 @@
                         "jp": "$.ig.DataSource からデータを要求するには、$.ig.DataSource により承諾されたリモート URL を指定します。"
                     }
                 },
-                "dataSourceType": {
```
Comment (bazal4o) [inline src/config/datachart.json:501]: If the user setup uses remote datasource with remote url, is this property required?

File: src/config/datachart.json [hunk 7]
```diff
@@ -498,22 +498,6 @@
...
-                    "type": "string",
-                    "defaultValue": null,
-                    "description": {
-                        "en": "Explicitly set data source type (such as \"json\"). Please refer to the documentation of $.ig.DataSource and its type property.",
-                        "jp": "データ ソースのタイプ (\"json\" など) を明示的に設定します。$.ig.DataSource とそのタイプ プロパティのドキュメントを参照してください。"
-                    }
-                },
-                "responseDataKey": {
```
Comment (bazal4o) [inline src/config/datachart.json:509]: If the user setup uses remote datasource with remote url, is this property required?

File: src/config/combo.json (hunk 3 above)
Comment (kdinev) [reply src/config/combo.json:623]: @bazal4o I believe all these should be configured only on the data source level in the designer (the non-visual control configuration) and then the data source should be assigned to the control. This simplifies the usage, as currently the control doesn't guide you on how these should be configured.

File: src/config/combo.json (hunk 4 above)
Comment (kdinev) [reply src/config/combo.json:615]: @bazal4o I believe all these should be configured only on the data source level in the designer (the non-visual control configuration) and then the data source should be assigned to the control. This simplifies the usage, as currently the control doesn't guide you on how these should be configured.

File: src/config/datachart.json (hunk 6 above)
Comment (kdinev) [reply src/config/datachart.json:501]: @bazal4o I believe all these should be configured only on the data source level in the designer (the non-visual control configuration) and then the data source should be assigned to the control. This simplifies the usage, as currently the control doesn't guide you on how these should be configured.

File: src/config/datachart.json (hunk 7 above)
Comment (kdinev) [reply src/config/datachart.json:509]: @bazal4o I believe all these should be configured only on the data source level in the designer (the non-visual control configuration) and then the data source should be assigned to the control. This simplifies the usage, as currently the control doesn't guide you on how these should be configured.

File: src/config/datachart.json [hunk 8]
```diff
@@ -1423,14 +1407,6 @@
...
         },
-        "dataSourceType": {
-            "type": "string",
-            "defaultValue": null,
-            "description": {
-                "en": "Explicitly set data source type (such as \"json\"). Please refer to the documentation of $.ig.DataSource and its type property.",
-                "jp": "データ ソースのタイプ (\"json\" など) を明示的に設定します。$.ig.DataSource とそのタイプ プロパティのドキュメントを参照してください。"
-            }
-        },
         "dataSourceUrl": {
```
Comment (bazal4o) [inline src/config/datachart.json:1410]: If we remove the response data key and all the related properties which can be configured via the DataSource, we need to remove dataSurceUrl property from the configuration.

**Review by bazal4o — APPROVED** [review]
I've added comments on some of the changes.

**Review by Lipata — APPROVED** [review]

**Pull request merged**